src/sources/file.rs

```diff
@@ -863,6 +881,7 @@ mod tests {
             },
             data_dir: Some(dir.path().to_path_buf()),
             glob_minimum_cooldown_ms: Duration::from_millis(100),
+            max_read_backoff_ms: Duration::from_millis(200),
```

Comment:
This is not the default. I think that relying on the `..Default::default()` that follows is fine here
```suggestion
```